### PR TITLE
Initial support for HT-VME290, sleep fixes for HT-VME213

### PR DESCRIPTION
**Pull request by todd-herbert**
* Initial support for Heltec Vision Master E290
* Decreased sleep current for Heltec Vision Master E213
  (~25µA deep sleep, ~15mA light sleep)

~~Note: this PR adjusts `doDeepSleep` for ESP32, please closely review https://github.com/meshtastic/firmware/commit/d41990b07052a2716d4de315a0a1e62eced78086~~.
Took the lazy way out: [aa6e45d](https://github.com/meshtastic/firmware/pull/4334/commits/aa6e45d8733d2d3a3a59431b038a9ba4cc6ae3e1)

Happy to let this one sit for a while if anyone is still waiting for hardware to arrive for testing.
___

Before merging:
- [x] Merge https://github.com/meshtastic/GxEPD2/pull/9 and update `lib_deps`
- [x] Remove `board_level = extra` from variant.h

Before next alpha:
- [ ] Add board to meshtastic/api repo

### CODE REVIEW

File: src/platform/esp32/main-esp32.cpp [hunk 1]
```diff
@@ -180,65 +180,21 @@ void esp32Loop()
...
-
-    // We want RTC peripherals to stay on
-    esp_sleep_pd_config(ESP_PD_DOMAIN_RTC_PERIPH, ESP_PD_OPTION_ON);
 
-    esp_sleep_enable_timer_wakeup(msecToWake * 1000ULL); // call expects usecs
-    esp_deep_sleep_start();                              // TBD mA sleep current (battery)
+    esp_sleep_pd_config(ESP_PD_DOMAIN_RTC_PERIPH, ESP_PD_OPTION_ON); // RTC must stay on for timer wakeup
+    esp_sleep_enable_timer_wakeup(msecToWake * 1000ULL);             // call expects usecs
+    esp_deep_sleep_start();
 }
```
Comment (todd-herbert) [inline src/platform/esp32/main-esp32.cpp:200]: Seeking particular feedback on these changes

Comment (todd-herbert) [reply src/platform/esp32/main-esp32.cpp]: Reverted these changes, really don't fancy breaking sleep for other variants. Instead, just excluded the pin 2 hold for the VME213 variant.

File: variants/heltec_wireless_paper_v1/variant.h [hunk 2]
```diff
@@ -16,9 +16,9 @@
 
 // SPI
 #define SPI_INTERFACES_COUNT 2
-#define PIN_SPI_MISO 10 // MISO
-#define PIN_SPI_MOSI 11 // MOSI
-#define PIN_SPI_SCK 9   // SCK
+#define PIN_SPI_MISO 11
```
Comment (thebentern) [inline variants/heltec_wireless_paper_v1/variant.h:19]: I thought the wireless paper variants were still working?

Comment (todd-herbert) [reply variants/heltec_wireless_paper_v1/variant.h:19]: Oh that's just tidying up those defines. Apparently they're not used, but they didn't actually match the [real values just below](https://github.com/meshtastic/firmware/blob/bcdda4de8ab81fec6a35be52cc2a3a0fa3fa5332/variants/heltec_wireless_paper/variant.h#L40-L43)
```cpp
#define LORA_SCK 9
#define LORA_MISO 11
#define LORA_MOSI 10
#define LORA_CS 8
```

Comment (thebentern) [reply variants/heltec_wireless_paper_v1/variant.h:19]: Oh! I was confused for a second 😅

**Review by thebentern — APPROVED** [review]

**Comment by Szetya** [comment]
I don't want to open a new issue for this. HT-VME is not built with the espressif32 6.7.0 platform. 
platform = platformio/espressif32@6.8.1 works.

message:
Using meshtastic platformio-custom.py, firmware version 2.4.3
Compiling .pio\build\heltec-vision-master-e290\src\BluetoothCommon.cpp.o
xtensa-esp32s3-elf-g++: error: CreateProcess: No such file or directory
Compiling .pio\build\heltec-vision-master-e290\src\ButtonThread.cpp.o
*** [.pio\build\heltec-vision-master-e290\src\BluetoothCommon.cpp.o] Error 1
xtensa-esp32s3-elf-g++: error: CreateProcess: No such file or directory
*** [.pio\build\heltec-vision-master-e290\src\ButtonThread.cpp.o] Error 1

**Pull request merged**